pkgs/applications/misc/osmscout-server/default.nix

```
@@ -0,0 +1,36 @@
+{ lib, mkDerivation, fetchFromGitHub, pkg-config
+, qmake, qttools, kirigami2, qtquickcontrols2, qtlocation
+, libosmscout, mapnik, valhalla, libpostal, osrm-backend
+, libmicrohttpd, sqlite, marisa, kyotocabinet, boost
+}:
+
+mkDerivation rec {
+  pname = "osmscout-server";
+  version = "1.17.1";
+
+  src = fetchFromGitHub {
+    owner = "rinigus";
+    repo = "osmscout-server";
+    rev = version;
+    sha256 = "0rpsi6nyhcz6bv0jab4vixkxhjmn84xi0q2xz15a097hn46cklx9";
+    fetchSubmodules = true;
+  };
+  patches = [ ./0001-Fix-MHD_-function-signatures-for-stricter-compilers.patch ];
```

Can you submit this patch to upstream?

```suggestion

  patches = [ ./0001-Fix-MHD_-function-signatures-for-stricter-compilers.patch ];
```

---

Yes... https://github.com/rinigus/osmscout-server/pull/350